### PR TITLE
bugfix: update dfgettask status to success

### LINKED ISSUE #792: bug: dfgettask should be updated to success status when finish downloading
### Ⅰ. Issue Description
Currently, when dfget download successfully, it will just exist. So this dfgettask in supernode will remain the status 'RUNNING' until it is deleted. I think this dfgettask should be updated to 'SUCCESS' status when the download is finished. 

### Ⅱ. Describe what happened
dfgettask in supernode will remain the status 'RUNNING' even if the download is successfully finished.

### Ⅲ. Describe what you expected to happen
Dfget will update the status of dfgettask to 'SUCCESS'.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅵ. Environment:

- dragonfly version:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### Ⅰ. Describe what this PR did
update dfget task to success

### Ⅱ. Does this pull request fix one issue?

fixes #792 

### Ⅳ. Describe how to verify it
Already verified in Prometheus
We can see the dfget task metrics with status "success" now
metrics

### CODE REVIEW

File: supernode/daemon/mgr/task/manager_util.go [hunk 1]
```diff
@@ -324,6 +324,12 @@ func (tm *Manager) parseAvailablePeers(ctx context.Context, clientID string, tas
 	pieceSuccess, _ := tm.progressMgr.GetPieceProgressByCID(ctx, task.ID, clientID, "success")
 	logrus.Debugf("taskID: %s, get successful pieces: %v", task.ID, pieceSuccess)
 	if cdnSuccess && (int32(len(pieceSuccess)) == task.PieceTotal) {
+		// update dfget task status to success
+		if err := tm.dfgetTaskMgr.UpdateStatus(ctx, clientID, task.ID, types.DfGetTaskStatusSUCCESS);
+			err != nil {
```
Comment (Starnop) [inline supernode/daemon/mgr/task/manager_util.go]: Don't enter a line break here. It's so weird.

Comment (yeya24) [reply supernode/daemon/mgr/task/manager_util.go]: This line is too long. IMO

Comment (yeya24) [reply supernode/daemon/mgr/task/manager_util.go]: Updated. PTAL.

**Comment by Starnop** [comment]
LGTM>

**Pull request merged**